lib/packages/core/src/burstService.ts

```diff
@@ -27,6 +27,7 @@ export class BurstService {
      * @param {string} relativePath The relative path will be prepended before each url created with toBRSEndpoint()
      * @param {Http?} httpClient If passed an client instance, it will be used instead of default HttpImpl. Good for testing.
      */
+    // TODO: introduce a Service Context class, substituting the parameter list by a single object
```

Comment:
using a context object allows to add additional features easier....in this case thought of introduce a kind of version checker, such we can be backwards compatible. In this specific case it is with CIP-19, which is only available since BRS 2.3.1